### PR TITLE
chore(transponder-configure-msg): update transponder configure message

### PR DESCRIPTION
**Pull request by suejungshin**
## Description of this change

Update ketch cli configure message

## Why is this change being made?
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How was this tested? How can the reviewer verify your testing?

Tested local

## Checklist

- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have evaluated the security impact of this change, and Secure Coding Practices have been observed.
- [x] I have informed stakeholders of my changes.

### CODE REVIEW

**Review by hanpeter — APPROVED** [review]
But I liked the blueberries and pancakes!

**Comment by hanpeter** [comment]
In more serious note, does it not print out that JSON error anymore?

**Review by smalexan — APPROVED** [review]
>:-(

File: pkg/transponder/configure.go [hunk 1]
```diff
@@ -100,9 +99,9 @@ func Configure(cmd *cobra.Command, args []string) error {
 	}
 
 	fmt.Println("Initiating database connection")
-	fmt.Println("Delivering blueberries and pancakes")
 
 	if resp.StatusCode == http.StatusNoContent {
+		fmt.Println("Successfully sent database connection request")
```
Comment (algorithmcardboard) [inline pkg/transponder/configure.go:104]: Can we say something like "Successfully configured the connection.  Please run the transponder list command to verify that the connection is active"

**Review by algorithmcardboard — APPROVED** [review]

**Comment by suejungshin** [comment]
> In more serious note, does it not print out that JSON error anymore?

Yeah, Sam fixed that with https://github.com/ketch-com/ketch-cli/pull/96

**Review by algorithmcardboard — COMMENTED** [review]
Looks good to me.

**Pull request merged**